napari/layers/base/base.py

```diff
@@ -11,6 +11,8 @@
 import magicgui as mgui
 import numpy as np
 
+from napari.types import FullLayerData
```

Comment:
can you guard this behind `if TYPE_CHECKING`

Reply:
Done